portfolio/src/main/webapp/index.html

```diff
@@ -48,18 +48,26 @@ <h3>More Pics of my Dogs!</h3>
         <a href="images/week8.JPG"><img src="images/week8.JPG"/></a>
     </div>
 
+
     <div id="content">
-      <h3>Comment Section</h3>
+        <h3>Comment Section</h3>
+        <label for="quantity">Show:</label>
+        <input type="number" id="num-choice" name="quantity" max="20" min="0">
+        <input type="submit" onclick="fetchComments()">
```

Comment:
optional: This works, but it requires an extra button press from the user. A less expensive method would be to add an onchange event handler to the number input element, and get rid of the submit button.